src/WebJobs.Extensions.EasyTables/Config/EasyTableHostConfigurationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Extension methods for EasyTable integration.
+    /// </summary>
+    public static class EasyTablesHostConfigurationExtensions
+    {
+        /// <summary>
+        /// Enables use of the EasyTable extensions.
+        /// </summary>
+        /// <param name="config">The <see cref="JobHostConfiguration"/> to configure.</param>
+        /// <param name="easyTablesConfig">The <see cref="EasyTableConfiguration"/> to use.</param>
+        [SuppressMessage("Microsoft.Design", "CA1011:ConsiderPassingBaseTypesAsParameters", Justification = "We explicitly need to register an EasyTableConfiguration.")]
```

Comment:
In this repo, I dump all of these suppressions in the global file. I know this is a personal preference thing.